includes/admin/class-wc-accommodation-booking-admin-panels.php

```diff
@@ -153,7 +153,7 @@ public function save_product_data( $post_id ) {
 		);
 
 		foreach ( $meta_to_save as $meta_key => $sanitize ) {
-			$value = ! empty( $_POST[ $meta_key ] ) ? $_POST[ $meta_key ] : '';
+			$value = sanitize_text_field( $_POST[ $meta_key ] ?? '' );
```

Comment:
I think you will also want to add a `wp_unslash` like so:

```suggestion
			$value = sanitize_text_field( wp_unslash( $_POST[ $meta_key ] ?? '' ) );
```

Reply:
Also, I am wondering if there should be a check to make sure that this booking does belong to the user (or that the user is an administrator).

Reply:
There are nonce and permission checks [right before](https://github.com/woocommerce/woocommerce/blob/94599a14cf5eb6efa5c706d3dc9bf491a756dbf4/plugins/woocommerce/includes/admin/class-wc-admin-meta-boxes.php#L268) the action is fired that this function is hooked to. That seems sufficient to me, but let me know if you disagree.

Reply:
makes sense, that looks sufficient